### PR TITLE
Add Deliveries notification handler

### PR DESCRIPTION
**Pull request by edymerchk**
## Explanation:

This PR includes the a Lita handler for deliveries notifications

TODO:

* maybe rename this repo?

### CODE REVIEW

**Review by ropiku — APPROVED** [review]

File: lib/lita/handlers/deliveries_notification_handler.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+require 'json'
+require 'httparty'
+
+module Lita
+  module Handlers
+    class DeliveriesNotificationHandler < Handler
+      SUBJECT_REGEX = /You have a delivery from/
+      UUID_REGEX = /[0-9a-f]{8}-([0-9a-f]{4}-){3}[0-9a-f]{12}/
+      PRODUCT_NAME = 'deliveries'
```
Comment (mattharris5) [inline lib/lita/handlers/deliveries_notification_handler.rb]: Should we `.freeze` these?

File: lib/lita/handlers/deliveries_notification_handler.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        }
+
+        HTTParty.post(
+          config.webhook_url,
+          body: payload.to_json,
+        )
+        response.reply(payload.to_json)
+      end
+    end
+    Lita.register_handler(DeliveriesNotificationHandler)
```
Comment (mattharris5) [inline lib/lita/handlers/deliveries_notification_handler.rb]: very minor thing, but maybe add a line break above this line to keep it separated from the class.

File: spec/lita/handlers/deliveries_notification_handler_spec.rb [hunk 3]
```diff
@@ -0,0 +1,28 @@
+require 'json'
+require 'spec_helper'
+
+describe Lita::Handlers::DeliveriesNotificationHandler, lita_handler: true do
+
+  before { allow(HTTParty).to receive(:post) }
+
+  it { is_expected.not_to route('How are you doing') }
+  it { is_expected.not_to route('To code or not to code, that is a question') }
+  it { is_expected.not_to route('How do you turn this one?') }
```
Comment (mattharris5) [inline spec/lita/handlers/deliveries_notification_handler_spec.rb]: Seems a bit excessive to have so many of these... ?

**Review by mattharris5 — APPROVED** [review]
Looks good - a couple of questions.

By the way, thank you for also updating the `README`!!

File: spec/lita/handlers/deliveries_notification_handler_spec.rb (hunk 3 above)
Comment (edymerchk) [reply spec/lita/handlers/deliveries_notification_handler_spec.rb]: hahaha its true, I just copy and paste from the other, but I will remove some 👍

File: lib/lita/handlers/deliveries_notification_handler.rb (hunk 2 above)
Comment (edymerchk) [reply lib/lita/handlers/deliveries_notification_handler.rb]: good idea, thanks 👍

File: lib/lita/handlers/deliveries_notification_handler.rb (hunk 1 above)
Comment (edymerchk) [reply lib/lita/handlers/deliveries_notification_handler.rb]: 👍

**Pull request merged**